app/build.gradle.kts

```diff
@@ -10,20 +10,21 @@ plugins {
 }
 
 android {
-    compileSdk = Versions.androidCompileSdkVersion
+    compileSdk = libs.versions.androidCompileSdkVersion.get().toInt()
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/kseito/RewardedTodo/blob/0a053f19cb2b422d8c2b69cb7cb9e8d853298ad8/app/build.gradle.kts#L13"></span>A newer version of <code>compileSdkVersion</code> than 35 is available: 36</td>
    </tr>
  </tbody>
</table>